src/DataBase/EntryDatabase.cs

```diff
@@ -160,12 +160,6 @@ public async Task<bool> DeleteListAsync(IEnumerable<int> idArray, int userId, En
 
         public async Task<IEntry> ReadAsync(int id, int userId, EntryType entryType)
         {
-            IEntryModel dbEntry = entryType switch
-            {
-                EntryType.Income => await Database.Incomes.FirstOrDefaultAsync(
-                    x => x.Id == id && x.UserId == userId),
-                _ => await Database.Expenses.FirstOrDefaultAsync(x => x.Id == id && x.UserId == userId)
-            };
```

Comment:
Why was this removed?

Reply:
This was unused code left by mistake actually, I only left the line used by the method